### PR TITLE
Add Signal Voltage Generation measurement

### PR DESCRIPTION
**Pull request by kirstyong**
### What does this Pull Request accomplish?

Adds the Signal Voltage Generation measurement for the PCBA-FCT Offering and generates a standard function of a sine and square wave as that is what the existing PCBA Toolkit currently supports.

### Why should this Pull Request be merged?

Measurement has been validated manually and built successfully.

### What testing has been done?

Connected Ch0 of FGEN to Ch0 of SCOPE.
image

**SCOPE reads back generated signal:**
image

**Generated square wave:**
image

**Measurement builds successfully:**
image

Built exe + other files copied to Services works.
image
image

TestStand
image

### CODE REVIEW

**Comment by kena0405** [comment]
@kirstyong I am not in the review list, but I saw your PR and found there's some missing VI (Wait Until Generation Complete.vi) in this branch. On both of the test machines, it is linked to the one `C:\Users\Administrator\Desktop\Wait Until Generation Complete.vi`. It seems it is part of Logic VI, so can you clarify if this is really in the correct directory?

File: src/NIFgenSignalVoltageGeneration/NIFgenSignalVoltageGeneration/Measurement Configuration.ctl
Comment (nathan-murphy) [inline src/NIFgenSignalVoltageGeneration/NIFgenSignalVoltageGeneration/Measurement Configuration.ctl:1]: It's not obvious that the Amplitude parameter is in Vpk-pk. Can you change the name of this control to make it more clear?

File: src/NIFgenSignalVoltageGeneration/NIFgenSignalVoltageGeneration/Measurement Logic.vi
Comment (nathan-murphy) [inline src/NIFgenSignalVoltageGeneration/NIFgenSignalVoltageGeneration/Measurement Logic.vi:1]: image
Can you set the `dt` of this waveform using the user-specified sample rate?

File: src/NIFgenSignalVoltageGeneration/NIFgenSignalVoltageGeneration/subVIs/Wait For Generation Complete.vi
Comment (nathan-murphy) [inline src/NIFgenSignalVoltageGeneration/NIFgenSignalVoltageGeneration/subVIs/Wait For Generation Complete.vi:1]: This VI opened way off screen for me. Can you please move it to top-left and re-save?

File: src/NIFgenSignalVoltageGeneration/NIFgenSignalVoltageGeneration/Measurement Logic.vi
Comment (nathan-murphy) [inline src/NIFgenSignalVoltageGeneration/NIFgenSignalVoltageGeneration/Measurement Logic.vi:1]: Actual Duration indicator is not used in the Measurement UI. Consider removing it from Logic, or adding to UI.

**Review by nathan-murphy — APPROVED** [review]
Minor comments. Fix it and ship it! 🛠 & 🚢

**Review by nburke-ni — APPROVED** [review]
Looks good to me

**Pull request merged**